tests/falco/commands_test.go

```diff
@@ -54,7 +54,7 @@ func TestFalco_Cmd_Version(t *testing.T) {
 		assert.NoError(t, res.Err(), "%s", res.Stderr())
 		assert.Equal(t, res.ExitCode(), 0)
 		assert.Regexp(t, regexp.MustCompile(
-			`Falco version:[\s]+[0-9]+\.[0-9]+\.[0-9](\-[0-9]+\+[a-f0-9]+)?[\s]+`+
+			`Falco version:[\s]+[0-9]+\.[0-9]+\.[0-9](\-[a-z0-9]+)?(\+[a-f0-9]+)?[\s]+`+
```

Comment:
```suggestion
			`Falco version:[\s]+(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?[\s]+`+
```

Why not use the officially suggested [SemVer2 regex](https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string)?

Reply:
Let's go for it!